### ISSUE TITLE
Model Training

### ISSUE THREAD

**Issue by githigher** [AIR-THU/DAIR-RCooper#16]
I noticed the training method you mentioned in your paper, “They are trained for 50 epochs with a batch size of 16”.
  Do you use multiple cards in your training? I only have 24G of video memory on one of my graphics cards, so there is no way to train more batch sizes with one graphics card. Can you offer a bit of training tips not mentioned in the paper? The experiments I've reproduced so far are still a bit off from your published data.
  Very much looking forward to your reply!

**Comment by leofansq** [comment]
Really glad that our work is valuable for you. We use one A100 (80G) card for each experiment. Due to the limitation of your GPU memory, you can adjust the learning rate and batch size correspondingly.
